frontend/src/metabase/meta/Parameter.js

```diff
@@ -25,9 +30,26 @@ export function getTemplateTagParameters(tags: TemplateTag[]): Parameter[] {
 export const getParametersBySlug = (parameters: Parameter[], parameterValues: ParameterValues): {[key:string]: string} => {
     let result = {};
     for (const parameter of parameters) {
-        if (parameterValues[parameter.id] !== undefined) {
+        if (parameterValues[parameter.id] != undefined) {
             result[parameter.slug] = parameterValues[parameter.id];
         }
     }
     return result;
 }
+
+export function getParameterTargetFieldId(target: ?ParameterTarget, datasetQuery: DatasetQuery): ?FieldId {
```

Comment:
Again, rather hard-to-read logic so short comment explaining the behavior would be nice